src/main/webapp/app/environments/environment.ts

```diff
@@ -0,0 +1,12 @@
+export const environment = {
+    firebaseConfig: {
```

Comment:
Why are we exposing again?

Reply:
We need to upload this file to heroku because heroku needs it to build the war file.